setup-configuration.md

```diff
@@ -53,6 +53,13 @@ There are small changes required to configure your site in Nginx.
 
 Use the following code in **server** section. If you have installed October into a subdirectory, replace the first `/` in location directives with the directory October was installed under:
 
+    # Basic CSP Configurations
+    add_header X-Content-Type-Options nosniff;
+    add_header X-XSS-Protection "1; mode=block";
+
+    # activate this rule if you want to enable HSTS
+    # add_header Strict-Transport-Security "max-age=31536000; includeSubDomains; preload" always;
+
```

Comment:
@mahony0 While I would like to document this sort of configuration somewhere in our docs, in the context of this section of the documentation, I don't think this is needed as we don't have security headers in any other webserver configuration. For the purpose of this PR, could you drop this? If you are interested in documenting the security headers that can be added, I would recommend doing them in their own section as another PR.
```suggestion
```

Reply:
Most of the people using this kind of configs out of the box so I think it would be useful if all in the same place. If you say so surely can remove them.